packages/client/src/helpers.ts

```diff
@@ -81,6 +89,24 @@ export async function executionInfoFromRaw<T>(
   };
 }
 
+export function decodeCountWorkflowExecutionsResponse(raw: RawCountWorkflowExecutions): CountWorkflowExecution {
+  return {
+    // Note: lossy conversion of Long to number
+    count: raw.count!.toNumber(),
+    groups: raw.groups!.map((group) => executionCountAggregationGroupFromRaw(group)),
+  };
+}
+
+export function executionCountAggregationGroupFromRaw(
```

Comment:
No need to export that function. And anyway, I would inline that function into `decodeCountWorkflowExecutionsResponse`.

Reply:
Done. I've inlined the AggregationGroup interface as well, we don't use it anymore.